src/components/Projects.tsx

```diff
@@ -16,7 +16,9 @@ export default function Projects({ projects, projectsRef }: Props): JSX.Element
       <p>
         Note: This list isn't up-to-date. Check my{' '}
         <Link href="https://github.com/JulienZD/">
-          <a className="link-animated-hover">Github</a>
+          <a className="link-animated-hover" target="_blank">
+            Github
```

Comment:
moet hier niet nog het <Github /> element in?

Reply:
bruh removed <Github />

Reply:
W HAT

Reply:
< Github />

Reply:
thanks

Reply:
No, this is just a link mentioning Github as text 
![image](https://github.com/JulienZD/website/assets/48630731/167a5070-7c81-4d4d-8ff8-6bcd68c7f698)